source/_components/cover.brunt.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: Brunt Cover
+description: "Instructions on how to set up Brunt Blind Engine within Home Assistant."
+date: 2018-06-24 14:36
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: brunt.png
+ha_category: Cover
+ha_release: 0.69
```

Comment:
`0.69` has already shipped a while ago, please update this to the next major release; `0.73`